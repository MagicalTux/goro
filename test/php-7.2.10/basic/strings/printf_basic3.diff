--
     int(6)
     
     -- Calling printf() with one arguments--
008- 11.110000
009- int(9)
010- 11.110000
011- int(9)
008+ 11.11000000000000
009+ int(17)
010+ 11.11000000000000
011+ int(17)
     
     -- Calling printf() with two arguments--
014- 11.110000 22.220000
015- int(19)
016- 11.110000 22.220000
017- int(19)
014+ 11.11000000000000 22.22000000000000
015+ int(35)
016+ 11.11000000000000 22.22000000000000
017+ int(35)
     
     -- Calling printf() with three arguments--
020- 11.110000 22.220000 33.330000
021- int(29)
022- 11.110000 22.220000 33.330000
023- int(29)
020+ 11.11000000000000 22.22000000000000 33.33000000000000
021+ int(53)
022+ 11.11000000000000 22.22000000000000 33.33000000000000
023+ int(53)
     ===DONE===
