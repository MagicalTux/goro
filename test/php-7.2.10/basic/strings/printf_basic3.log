
---- EXPECTED OUTPUT
*** Testing printf() : basic functionality - using float format ***

-- Calling printf() with no arguments--
format
int(6)

-- Calling printf() with one arguments--
11.110000
int(9)
11.110000
int(9)

-- Calling printf() with two arguments--
11.110000 22.220000
int(19)
11.110000 22.220000
int(19)

-- Calling printf() with three arguments--
11.110000 22.220000 33.330000
int(29)
11.110000 22.220000 33.330000
int(29)
===DONE===
---- ACTUAL OUTPUT
*** Testing printf() : basic functionality - using float format ***

-- Calling printf() with no arguments--
format
int(6)

-- Calling printf() with one arguments--
11.11000000000000
int(17)
11.11000000000000
int(17)

-- Calling printf() with two arguments--
11.11000000000000 22.22000000000000
int(35)
11.11000000000000 22.22000000000000
int(35)

-- Calling printf() with three arguments--
11.11000000000000 22.22000000000000 33.33000000000000
int(53)
11.11000000000000 22.22000000000000 33.33000000000000
int(53)
===DONE===
---- FAILED
