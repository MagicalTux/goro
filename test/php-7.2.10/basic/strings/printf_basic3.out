*** Testing printf() : basic functionality - using float format ***

-- Calling printf() with no arguments--
format
int(6)

-- Calling printf() with one arguments--
11.11000000000000
int(17)
11.11000000000000
int(17)

-- Calling printf() with two arguments--
11.11000000000000 22.22000000000000
int(35)
11.11000000000000 22.22000000000000
int(35)

-- Calling printf() with three arguments--
11.11000000000000 22.22000000000000 33.33000000000000
int(53)
11.11000000000000 22.22000000000000 33.33000000000000
int(53)
===DONE===